Appraisals

```diff
@@ -1,5 +1,5 @@
 appraise "sass-3-4" do
-  gem "sass", "~> 3.4"
+  gem "sass", "~> 3.4.22"
```

Comment:
FYI, I opened https://github.com/thoughtbot/bourbon/pull/1015 on Bourbon to loosen these requirements.